### DISCUSSION TITLE
Image Optimization causes out of memory error

### DISCUSSION THREAD

**Discussion by ava-gordon** [vercel/next.js#24551] [Help]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

14.16.1

### What browser are you using?

Chrome

### What operating system are you using?

ubuntu

### How are you deploying your application?

next start

### Describe the Bug

I'm running a nextjs app on a t2.micro AWS instance, and using next/image to serve an optimized version of a cover photo for my site's splash page. However, the node process is getting killed every time it tries to serve the image; it serves the rest of the site just fine, but trying to serve my single optimized image results in the node server getting killed with an error code 137 (an out of memory error).  Inspecting with htop reveals that the node process is pegging the CPU to 100% just before it gets killed. Of course, one solution would be to use a larger instance with more memory, but nobody is using this site yet so I'd prefer to keep costs down until the site is actually live. Besides, I'm not clear why next should need so much CPU and memory to optimize an image that is 7mb in size. Removing the optimization and serving the un-optimized image prevents this issue from happening, but obviously I'd prefer to use the optimized images in prod wherever possible.

### Expected Behavior

Image optimization features do not result in the linux process manager killing my app for using too much memory.

### To Reproduce

Try to use next/image to serve any large jpeg on a t2.micro AWS instance (or any other machine instance with limited memory).

**Comment by grgmgd** [comment]
I am facing the same issue using a DigitalOcean 1GB memory droplet
